ci-operator/config/openshift/ingress-node-firewall/openshift-ingress-node-firewall-master.yaml

```diff
@@ -34,13 +41,11 @@ tag_specification:
   name: "4.12"
   namespace: ocp
 tests:
-- as: e2e-aws
-  steps:
-    cluster_profile: aws
-    workflow: openshift-e2e-aws
-- as: ingress-node-firewall-e2e-metal-ipi
-  steps:
-    cluster_profile: packet
+- as: test-fmt
```

Comment:
Is this replacing e2e tests with linter tests?

Reply:
we are working on bringing in the e2e test we have today on kind to run here in separate PRs till we have all OCP's bits in place